src/components/molecules/FormFields/URLInput/Input.tsx

```diff
@@ -30,7 +32,7 @@ export default function FileInput({
         onClick={(e: React.SyntheticEvent) => e.preventDefault()}
         disabled={!field.value}
       >
-        {isLoading ? <Loader /> : 'Add File'}
+        {isLoading ? <Loader /> : `${submitText}`}
```

Comment:
too much template literals wrapping, this should be enough if submitText is really always a string:

```suggestion
        {isLoading ? <Loader /> : submitText}
```

Reply:
Thanks for the suggestion, this is fixed now